### DISCUSSION TITLE
如何将vue组件打包成npm 包，以提供给不同的分包分发组件

### DISCUSSION THREAD

**Discussion by tu6ge** [NervJS/taro#9873] [Q & A]
如何将vue组件打包成npm 包，以提供给不同的分包分发组件

**Comment by tu6ge** [comment]
小程序页面用引入组件的方式引入

**Reply by tu6ge** [reply]
有了解这方便的 coder ，给提供一个参考文档就行

**Reply by shichaohui** [reply]
大佬有解决 vue 组件发布 npm 的问题吗？马上 2023 年了，还是很少这方面的资料。

**Reply by yoyo837** [reply]
不太理解这和Taro有什么关系, 自己发布npm就行了呀.

**Reply by shichaohui** [reply]
不清楚 taro 里面的 vue 组件要怎么打包才能兼容各端，使用 rollup 打包需要做哪些配置。
